### PR TITLE
Handle stack removal failures

### PR DESCRIPTION
**Pull request by perrin4869**
Haven't tested if this works yet, is this what you meant? @medikoo 
I'll test it later if it works!
Also I went ahead and removed `bluebird`, don't think anyone needs it anymore...

### CODE REVIEW

**Comment by perrin4869** [comment]
Closes #10

**Review by medikoo — COMMENTED** [review]
@perrin4869 it looks great, still can you make different PR with blubird removal? Ideally, if two different things are not mixed under one PR (if one raises issues other thing is not blocked)

**Comment by perrin4869** [comment]
Of course, it's a bad habit of mine haha
Done!

File: index.js [hunk 1]
```diff
@@ -13,7 +13,9 @@ class ServerlessPluginVpcEniCleanup {
 		this.provider = this.serverless.getProvider(this.serverless.service.provider.name);
 		this.hooks = {
 			"before:remove:remove": this.cleanup.bind(this),
-			"after:remove:remove": () => (this.isDisabled = true)
+			"after:remove:remove": () => (this.isDisabled = true),
+			// for example, if the stack does not exist
+			"error:remove:remove": () => (this.isDisabled = true)
```
Comment (medikoo) [inline index.js]: It should be just `error` :)

**Review by medikoo — CHANGES_REQUESTED** [review]

File: index.js (hunk 1 above)
Comment (perrin4869) [reply index.js]: Thanks!

**Review by medikoo — APPROVED** [review]
Thank you!

**Pull request merged**